sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -505,7 +506,7 @@ async def _read(self, n, buffer=None, **kwargs):  # pylint: disable=unused-argum
 
         try:
             while n:
-                data = await self.ws.receive_bytes()
+                data = await self.ws.receive_bytes(timeout=self._connect_timeout * 2)
```

Comment:
Could we put a big inline comment describing this (and the heartbeat config above) - maybe with a link to the open aiohttp issue that we are waiting on to be resolved?

Reply:
Added in a block comment and remove this particular mitigation. Receives tend to time out rather quickly as it is, so no need to constrain it manually.